### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sarah-svg**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,5 +1,83 @@
...
 
-// initialize state
 
-// set event listeners to update state and DOM
\ No newline at end of file
+for (let i = 0 ; i < buttons.length ; i++){
+    buttons[i].addEventListener('click', getRandomThrow);
+}
+
+function getRandomThrow(e){
```
Comment (breeannb) [inline app.js:11]: great job!

File: test/example.test.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ test('time to test a function', (expect) => {
```
Comment (breeannb) [inline test/example.test.js:10]: need to TDD your functions for getRandomThrow and didUserWin

Comment (sarah-svg) [reply test/example.test.js:10]: yes I have not got to that was going to try to get to that today..

Comment (sarah-svg) [reply test/example.test.js:10]: thank you Breeann always looking for you input

**Pull request merged**